Core.UnitTests/MutableReflection/Implementation/MethodBaseDefinitionCacheTest.cs

```diff
@@ -29,19 +29,21 @@ public class MethodBaseDefinitionCacheTest
     [Test]
     public void GetBaseDefinition_ForCustomMethodInfo_NotCached ()
     {
-      var customMethodInfoMock = MockRepository.GenerateStrictMock<CustomMethodInfo> (
-          ReflectionObjectMother.GetSomeType(), "method", MethodAttributes.Public, null, Type.EmptyTypes);
+      var customMethodInfoMock = new Mock<CustomMethodInfo> (
+          MockBehavior.Strict, ReflectionObjectMother.GetSomeType(), "method", MethodAttributes.Public, null, Type.EmptyTypes);
 
       var fakeMethodDefinition1 = ReflectionObjectMother.GetSomeMethod();
-      customMethodInfoMock.Expect (mock => mock.GetBaseDefinition ()).Return (fakeMethodDefinition1).Repeat.Once();
-
       var fakeMethodDefinition2 = ReflectionObjectMother.GetSomeMethod();
-      customMethodInfoMock.Expect (mock => mock.GetBaseDefinition ()).Return (fakeMethodDefinition2).Repeat.Once ();
 
-      var result1 = MethodBaseDefinitionCache.GetBaseDefinition (customMethodInfoMock);
-      var result2 = MethodBaseDefinitionCache.GetBaseDefinition (customMethodInfoMock);
+      customMethodInfoMock
+          .SetupSequence (mock => mock.GetBaseDefinition ())
```

Comment:
After playing around with it, to my surprise, it looks like it's **not**. SetupSequence seems to be only usable for stubbing, if we setup 2 different return values and the method is only called once it will **not** fail. A separate `MockSequence` object **does** have an implicit `Verifiable()`, though. `SetupSequence` also doesn't even offer `Verifiable()` on its return type, so there's that...

I'll take a look at all occurrences of `SetupSequence` and make sure to rewrite them so they verify correctly